TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Status.java

```diff
@@ -41,12 +42,14 @@
     }};
     public static final double BUCKET_WAIT_TIME = 0.2;
     public static final double PITSTOP_WAIT_TIME = 0.15;
+    public static final double OUTRIGGER_UP = 1.0;
+    public static final double OUTRIGGER_DOWN = 0.0;
```

Comment:
Refactor into Hashmap to keep consistent with other sub-assemblies

Reply:
Okay, I will do that.